tests/files/bugs.parse

```diff
@@ -39,3 +39,5 @@ TS: 0 | 1997-01-01 -99999:-99999:-99999  0Y   0M  -2D /   0H   0M   0S | 1997W01
 TS: 0 | 2004-01-01 05:00:00 GMT 00000  0Y   0M  60D /   0H   0M   0S | 2004W101T05:00+0
 TS: 0 | -99999--99999--99999 -99999:-99999:-99999 | nextyear
 TS: 0 | -99999--99999--99999 -99999:-99999:-99999  1Y   0M   0D /   0H   0M   0S | next year
+TS: 0 | 10000-01-01 00:00:00 | 10000-01-01T00:00:00
+TS: 0 | -10000-01-01 00:00:00 | -10000-01-01T00:00:00
```

Comment:
I would add a few more, for example with the year 99999 and 100000 (+/-), and a few with year numbers that are really big - to test the upper year range (Similar to what you did in https://github.com/php/php-src/pull/3380#pullrequestreview-143656500 ) — and test the maximum length of "20" characters too.